go/vt/vtgate/vtgate.go

```diff
@@ -132,6 +134,7 @@ func registerFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&enableSchemaChangeSignal, "schema_change_signal", enableSchemaChangeSignal, "Enable the schema tracker; requires queryserver-config-schema-change-signal to be enabled on the underlying vttablets for this to work")
 	fs.StringVar(&schemaChangeUser, "schema_change_signal_user", schemaChangeUser, "User to be used to send down query to vttablet to retrieve schema changes")
 	fs.IntVar(&queryTimeout, "query-timeout", queryTimeout, "Sets the default query timeout (in ms). Can be overridden by session variable (query_timeout) or comment directive (QUERY_TIMEOUT_MS)")
+	fs.BoolVar(&vstreamCellAliasFallback, "vstream_cell_alias_fallback", vstreamCellAliasFallback, "Determines whether to default to cell aliases for VStream tablet selection instead of having the gRPC request specify this in the VStreamrRequest object.")
```

Comment:
I think this would be correct and more clear:
```
"By default, when no tablets can be found in the local cell for streaming, also look for tablets in a cell alias which includes the local cell")
```

As the gRPC cells I'm assuming will be used when specified and this flag then has no impact. So it only alters the default behavior when no cells are provided and searches the alias as a fallback if needed.